keyboards/draculad/rules.mk

```diff
@@ -6,15 +6,16 @@ BOOTLOADER = caterina
 
 # Build Options
 #   change yes to no to disable
-#
+
+VIA_ENABLE = yes
 BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
-MOUSEKEY_ENABLE = yes       # Mouse keys
+MOUSEKEY_ENABLE = no       # Mouse keys
 EXTRAKEY_ENABLE = yes       # Audio control and System control
 CONSOLE_ENABLE = no         # Console for debug
 COMMAND_ENABLE = no         # Commands for debug and configuration
 NKRO_ENABLE = yes           # Enable N-Key Rollover
 BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
-RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+RGBLIGHT_ENABLE = no       # Enable keyboard RGB underglow
```

Comment:
Don't change the root rules.mk, instead add all the changes as overrides to the vial keymap's rules.mk

Reply:
> Don't change the root rules.mk, instead add all the changes as overrides to the vial keymap's rules.mk

Have changed the file back and resubmitted the commit.